.github/workflows/upgrade_downgrade_test_backups_e2e.yml

```diff
@@ -10,33 +10,10 @@ concurrency:
 permissions: read-all
 
 jobs:
-  get_previous_release:
-    if: always()
-    name: Get Previous Release - Backups - E2E
-    runs-on: gh-hosted-runners-16cores-1
-    outputs:
-      previous_release: ${{ steps.output-previous-release-ref.outputs.previous_release_ref }}
-
-    steps:
-      - name: Check out to HEAD
-        uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-
-      - name: Set output with latest release branch
-        id: output-previous-release-ref
-        run: |
-          previous_release_ref=$(./tools/get_previous_release.sh ${{github.base_ref}} ${{github.ref}})
-          echo $previous_release_ref
-          echo "previous_release_ref=${previous_release_ref}" >> $GITHUB_OUTPUT
-
   upgrade_downgrade_test_e2e:
     timeout-minutes: 60
-    if: always() && needs.get_previous_release.result == 'success'
     name: Run Upgrade Downgrade Test - Backups - E2E
```

Comment:
nit: we could rename this to have the same name as the workflow name in order to have a cleaner list on the PR page: 
```
Backups - E2E - Next Release - Upgrade Downgrade Testing / Run Backups - E2E - Next Release - Upgrade Downgrade Testing
```
instead of
```
Backups - E2E - Next Release - Upgrade Downgrade Testing / Run Upgrade Downgrade Test - Backups - E2E - Next Release
```

Reply:
I know, I thought of this, but I didn't do it, because it would require changing all the branch protection rules, which I kinda wanted to avoid. (Just additional work)